src/Site.js

```diff
@@ -1,5 +1,4 @@
 const cheerio = require('cheerio');
-const ejs = require('ejs');
```

Comment:
Can we remove the ejs package from `package.json` & `package-lock.json` as well?

Reply:
Just did that. Thanks for reminding!